src/daemon/jsonrpc/api.rs

```diff
@@ -221,31 +244,68 @@ macro_rules! parse_vault_status {
     };
 }
 
+macro_rules! internal_error {
+    ($error: expr) => {
+        JsonRpcError {
+            code: jsonrpc_core::types::error::ErrorCode::InternalError,
+            message: format!("Internal error: {:?}", $error),
```

Comment:
nit: i think it's awkward for the user to receive, eg, "Internal error: cosigning server error". Can we keep the placeholder code until we have some dedicated ones but drop the `Internal error` prefix?